packages/generator/src/generator-without-ts-morph/service/file.ts

```diff
@@ -18,9 +18,7 @@ ${serviceClass(service)}`;
 }
 
 function getImports(service: VdmServiceMetadata): Import[] {
-  const operationImports = service.functionImports.concat(
-    service.actionImports || []
-  );
+  const operationImports = service.operationImports;
   if (operationImports.length === 0) {
```

Comment:
[req] I know this was here before, but it does not match our style guide: https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md#use-truthyfalsy-checks-where-possible

```suggestion
  if (!operationImports.length) {
```